### PR TITLE
`evaluate` method for `WelfordOp`

### PR DESCRIPTION
**Pull request by Priya2698**
Issue #670

### CODE REVIEW

File: third_party/googletest
Comment (zasdfgbnm) [inline third_party/googletest:1]: Is this change accidental?

**Review by zasdfgbnm — APPROVED** [review]

File: third_party/googletest
Comment (Priya2698) [reply third_party/googletest:1]: Yes, fixed now.

**Comment by Priya2698** [comment]
!build

**Comment by Priya2698** [comment]
!build

**Pull request merged**